src/sass/helpers/_message_bubble.scss

```diff
@@ -3,16 +3,16 @@ $message-bubble-color-out: #dff0d8;
 $bubble-triangle-size: $main-padding;
 
 @mixin message-bubble-shadow {
-  box-shadow: 0 1px 1px rgba(0,0,0,0.25);
+    box-shadow: 0 1px 1px rgba(0, 0, 0, .25);
 }
 
 @mixin message-bubble-triangle {
-  content: "";
-  position: absolute;
-  border-radius: 1px;
-  transform: rotate(-45deg);
-  width: $bubble-triangle-size;
-  height: $bubble-triangle-size;
-  left: ($bubble-triangle-size/2);
-  top: $bubble-triangle-size;
-}
\ No newline at end of file
+    position: absolute;
+    top: $bubble-triangle-size;
+    left: ($bubble-triangle-size / 2);
```

Comment:
Redundant parentheses. Is there no lint for that? :slightly_smiling_face:

Reply:
I guess not.